Classes/Indexer/NodeIndexer.php

```diff
@@ -270,10 +274,13 @@ public function indexNode(Node $node, ?WorkspaceName $targetWorkspaceName = null
             }
         };
 
-        $handleNode = function (Node $node, WorkspaceName $workspaceName, DimensionSpacePoint $dimensionSpacePoint) use ($contentRepository, $targetWorkspaceName, $indexer) {
-            $subgraph = $contentRepository->getContentGraph($workspaceName)->getSubgraph($dimensionSpacePoint, VisibilityConstraints::withoutRestrictions());
-            $nodeFromContext = $subgraph->findNodeById($node->aggregateId);
-
+        $handleNode = function (Node $node, WorkspaceName $workspaceName, DimensionSpacePoint $dimensionSpacePoint) use ($contentRepository, $targetWorkspaceName, $indexer, &$nodeFromContext) {
+            $nodeFromContext = $this->securityContext->withoutAuthorizationChecks(
+                function () use ($contentRepository, $node, $workspaceName, $dimensionSpacePoint) {
+                    $subgraph = $contentRepository->getContentGraph($workspaceName)->getSubgraph($dimensionSpacePoint, VisibilityConstraints::withoutRestrictions());
+                    return $subgraph->findNodeById($node->aggregateId);
+                }
+            );
```

Comment:
Another cosmetic tweak, but IMO this is a bit easier to read:
```suggestion
            $nodeFromContext = $this->securityContext->withoutAuthorizationChecks(
                fn () => $contentRepository
                    ->getContentGraph($workspaceName)
                    ->getSubgraph($dimensionSpacePoint, VisibilityConstraints::withoutRestrictions())
                    ->findNodeById($node->aggregateId);
            );
```

Reply:
(untested though)

Reply:
Done. Thank you!